web/src/components/studies/CreateStudyForm.tsx

```diff
@@ -0,0 +1,371 @@
+            <input type="checkbox" id="hra" {...register("hra")} />{" "}
+            <span>
+              We will be seeking/have sought{" "}
+              <a href="https://www.hra.nhs.uk/approvals-amendments/what-approvals-do-i-need/hra-approval/">
+                Health Research Authority
+              </a>{" "}
+              approval of this research
+            </span>
+          </Label>
+          {showIrasId && (
+            <Label htmlFor="irasId">
+              <span>
+                {" "}
+                <a href="https://www.gov.uk/guidance/clinical-trials-for-medicines-apply-for-approval-in-the-uk#:~:text=Integrated%20Research%20Application%20System%20(IRAS)">
+                  IRAS
+                </a>{" "}
+                ID (if applicable)
+              </span>
+
+              <Input type="text" id="irasId" {...register("irasId")} />
+            </Label>
+          )}
+        </fieldset>
+        <fieldset className={getFieldsetClass(2)}>
+          <legend>NHS</legend>
+
+          <Label htmlFor="nhs" className={styles["checkbox-label"]}>
+            <input type="checkbox" id="nhs" {...register("nhs")} />
+            This research is associated with the NHS, uses NHS data, works with NHS sites or has use of a/some NHS
+            facilities
+          </Label>
+          {showNhsRelated && (
+            <>
+              <Label htmlFor="nhsEngland" className={styles["checkbox-label"]}>
+                <input type="checkbox" id="nhsEngland" {...register("nhsEngland")} />
+                NHS England will be involved in gatekeeping and/or providing data for this research
+              </Label>
+              {showNhsEnglandRef && (
+                <Label htmlFor="nhsEnglandRef">
+                  NHS England NIC number (if applicable)
+                  <Input type="number" id="nhsEnglandRef" {...register("nhsEnglandRef")} />
+                </Label>
+              )}
+
+              <Label htmlFor="mnca" className={styles["checkbox-label"]}>
+                <input type="checkbox" id="mnca" {...register("mnca")} />{" "}
+                <span>
+                  The{" "}
+                  <a href="https://www.myresearchproject.org.uk/help/hlptemplatesfor.aspx">
+                    HRA Model Non-Commercial Agreement
+                  </a>{" "}
+                  will be in place across all sites when working with NHS sites
+                </span>
+              </Label>
+              <Label htmlFor="dspt" className={styles["checkbox-label"]}>
+                <input type="checkbox" id="dspt" {...register("dspt")} />
+                This research requires an NHS Data Security & Protection Toolkit registration to be in place at UCL.
+                (This might arise when approaching public bodies for NHS and social care data)
+              </Label>
+            </>
+          )}
+        </fieldset>
+
+        {/* third step */}
+        <fieldset className={getFieldsetClass(3)}>
```

Comment:
on this step the progress indicator isn't quite right(?)

<img width="663" alt="Screenshot 2025-07-08 at 12 22 30" src="https://github.com/user-attachments/assets/8d6ae8a9-f5fb-4ab2-8a27-4c52d7048b67" />